spec/models/answer_spec.rb

```diff
@@ -0,0 +1,92 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Answer, type: :model do
+  it 'is valid if all attributes are present and belongs to a question' do
+    quiz = Quiz.create(title: 'test')
+    question = Question.create(
+      answer_type: '1',
```

Comment:
It is a bit weird that you are passing `String`s here, even though the column types are defined as `Integers`. Rails will automatically convert things for you in this case, but it would be better if you can keep consistency in the code!

Reply:
Thank you!

Yeah, I asked Sorin about that but I guess he missed the question :D. I was not sure about it in the end. I will make the changes.